apps/teamsjs-diagnostic-app/public/auth-end.html

```diff
@@ -0,0 +1,59 @@
+<!--This file is used during the Teams authentication flow to assist with retrieval of the access token.-->
+<!--If you're not familiar with this, do not alter or remove this file from your project.-->
+<html>
+  <head>
+    <title>Login End Page</title>
+    <meta charset="utf-8" />
+  </head>
+
+  <body>
+    <script
+      src="https://res.cdn.office.net/teams-js/2.22.0/js/MicrosoftTeams.min.js"
```

Comment:
Would we have to manually update this script html for every new CDN release of TeamsJS? I might just create a work item to add modifying this file to the prerelease script for myself in the future.